### ISSUE TITLE
is it possible to adapt for wii u version?

### ISSUE THREAD

**Issue by Spaun39** [LITTOMA/vfs2#2]
wiiu vfs https://drive.google.com/file/d/1bTlY7FpXzeDiqSMbZe1lH5vLEUcIefoJ/view?usp=sharing

**Comment by LITTOMA** [comment]
not sure. I'll check it out if I have time.

**Comment by Spaun39** [comment]
thank you

**Comment by boreals-back-again** [comment]
@Spaun39 I'm willing to wager that the only difference between this and the Wii U format is the endianness, as the Wii U was PowerPC (BE) to the Vita's ARM (LE).

**Comment by Spaun39** [comment]
@boreals-back-again Sounds plausible. I'd like to take the time to figure it out.

**Comment by boreals-back-again** [comment]
> @boreals-back-again Sounds plausible. I'd like to take the time to figure it out.

Actually I opened it with a hex editor and the file looks little endian to me. Did you try running the script? It looked like Python 2 to me

**Comment by Spaun39** [comment]
@boreals-back-again Of course, otherwise I would have modified the game long ago. I don't remember exactly how it didn't work; it either froze or there was an error. My memory is bad.
Maybe the markup is different or the headings have some differences, I would like to check this, but I can’t do it now

**Comment by boreals-back-again** [comment]
> @boreals-back-again Of course, otherwise I would have modified the game long ago. I don't remember exactly how it didn't work; it either froze or there was an error. My memory is bad.
> Maybe the markup is different or the headings have some differences, I would like to check this, but I can’t do it now

what's ur discord

**Comment by Spaun39** [comment]
@boreals-back-again 
W_wex#9962

**Comment by boreals-back-again** [comment]
> [@boreals-back-again](https://github.com/boreals-back-again) W_wex#9962

Sorry ive been kind of busy lately so it took me a while to add you

**Comment by LITTOMA** [comment]
Did you guys made any progress on this? I just forgot this issue. Sorry.
Adding supports for the wiiu version could be much easier since we have AI programming nowadays :)
